dependencies.yaml

```
@@ -370,6 +377,21 @@ dependencies:
           # Align nvcomp version with rapids-cmake
           - nvcomp==4.0.1
           - spdlog>=1.14.1,<1.15
+  depends_on_nvcomp:
+    specific:
```

Let's use this dependency list to specify the conda package, too.
```suggestion
    common:
      - output_types: conda
        packages:
          - nvcomp==4.0.1
    specific:
```